packages/web/src/components/typography/typography.module.css

```diff
@@ -17,91 +17,169 @@
  ```
 */
 
+/* Display Styles */
+
+/* 96 / 120 */
+.diplayXLarge {
+  font-size: var(--unit-24);
+  font-weight: var(--font-bold);
+  line-height: calc(var(--unit) * 30);
+}
+/* 72 / 88 */
+.displayLarge {
+  font-size: var(--unit-18);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-22);
+}
+/* 56 / 68 */
+.displayMedium {
+  font-size: var(--unit-14);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-17);
+}
+/* 42 / 52 */
+.displaySmall {
+  font-size: calc(var(--unit) * 10.5);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-13);
+}
+.displayStrong {
+  font-weight: var(--font-heavy);
+}
+
+/* Heading Styles */
+
+/* 36 / 40 */
+.headingXLarge {
+  font-size: var(--unit-9);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-10);
+}
+/* 28 / 32 */
+.headingLarge {
+  font-size: var(--unit-7);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-8);
+}
+/* 24 / 32 */
+.headingMedium {
+  font-size: var(--unit-6);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-8);
+}
+/* 20 / 24 */
+.headingSmall {
+  font-size: var(--unit-5);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-6);
+}
+.headingStrong {
+  font-weight: var(--font-heavy);
+}
+
 /* Title Styles */
+
+/* 18 / 24 */
 .titleLarge {
-  font-size: var(--font-l);
+  font-size: calc(var(--unit) * 4.5);
   font-weight: var(--font-bold);
   line-height: var(--unit-6);
 }
+/* 16 / 20 */
 .titleMedium {
-  font-size: var(--font-m);
+  font-size: var(--unit-4);
   font-weight: var(--font-bold);
   line-height: var(--unit-5);
 }
+/* 14 / 16 */
 .titleSmall {
-  font-size: var(--font-s);
+  font-size: calc(var(--unit) * 3.5);
   font-weight: var(--font-bold);
   line-height: var(--unit-4);
 }
+/* 12 / 16 */
 .titleXSmall {
-  font-size: var(--font-xs);
+  font-size: var(--unit-3);
   font-weight: var(--font-bold);
   line-height: var(--unit-4);
 }
-
-.titleLight {
+.titleWeak {
   font-weight: var(--font-demi-bold);
 }
 .titleStrong {
   font-weight: var(--font-heavy);
 }
 
-/* Body Styles */
-.bodyLarge {
-  font-size: var(--font-l);
-  font-weight: var(--font-medium);
-  line-height: var(--unit-6);
-}
-.bodyMedium {
-  font-size: var(--font-m);
-  font-weight: var(--font-medium);
-  line-height: var(--unit-5);
-}
-.bodySmall {
-  font-size: var(--font-s);
-  font-weight: var(--font-medium);
-  line-height: var(--unit-5);
-}
-.bodyXSmall {
-  font-size: var(--font-xs);
-  font-weight: var(--font-medium);
-  line-height: var(--unit-5);
-}
-
-.bodyStrong {
-  font-weight: var(--font-demi-bold);
-}
-
 /* Label Styles */
 
+/* 20 / 24 */
 .labelXLarge {
-  font-size: var(--font-xl);
+  font-size: var(--unit-5);
```

Comment:
Note on this: I think we want the typography components/modules to replace our usage of the font size CSS variables long term? So I opted to change these so they derive from the grid units directly, instead of using the font size variables.

Reply:
Stellar job and makes sense using the units.